heartbeat/oracle

```diff
@@ -439,7 +441,13 @@ check_mon_user() {
 			ocf_log err "sqlplus output: $output( $output2 )"
 			return 1
 		fi
+	else
```

Comment:
IIUC, an existing setup with existing user profile would have already returned from above and continued in oracle_start() no matter what.

When it's coming to here, it means the user profile doesn't exist and it's on its way to create the user profile in below. So it seems like a right place to check what `OCF_RESKEY_monpassword` is like.

Not necessary to change. It'd be the same without `else`, but it'd probably be clearer ...

Reply:
It's set to `OCFMON` from $OCF_RESKEY_monpassword_default on the following line when monpassword isnt specified by the user:
https://github.com/ClusterLabs/resource-agents/blob/f68497ff92b42e986ddd56e8f0190f9ce7633a06/heartbeat/oracle#L779

Reply:
Thanks for the review!

I moved the check out of the whole block for a better readability.